src/BuildIt.Forms/BuildIt.Forms.Controls/BuildIt.Forms.Controls.csproj

```diff
@@ -83,7 +83,17 @@
     </PackageReference>
   </ItemGroup>
 
+  <ItemGroup>
+    <EmbeddedResource Update="PullToRefreshControl.xaml">
+      <Generator>MSBuild:UpdateDesignTimeXaml</Generator>
+    </EmbeddedResource>
+  </ItemGroup>
+
   <ItemGroup>
     <PackageReference Update="StyleCop.Analyzers" Version="1.1.118" />
   </ItemGroup>
+
+  <ItemGroup>
```

Comment:
This shouldn't be here - I think the renderes folder is include by default for android builds

Reply:
Removed this and other XAML file entry.